resources/page/permalinks.go

```diff
@@ -263,3 +272,80 @@ func (l PermalinkExpander) pageToPermalinkSection(p Page, _ string) (string, err
 func (l PermalinkExpander) pageToPermalinkSections(p Page, _ string) (string, error) {
 	return p.CurrentSection().SectionsPath(), nil
 }
+
+var (
+	nilSliceFunc = func(s []string) []string {
+		return nil
+	}
+	allSliceFunc = func(s []string) []string {
+		return s
+	}
+)
+
+func (l PermalinkExpander) toSliceFunc(cut string) func(s []string) []string {
+	cut = strings.ToLower(strings.TrimSpace(cut))
```

Comment:
`strings.ToLower` can happen after these two if-statements.

Reply:
Agree, but ref. my comment above, this isn't a hot path. We set this up once and reuse them.